ushahidi/platform
# Adding language-switch to api + translating dataprovider-texts

This pull request makes the following changes:
- Sets the language in Kohana to the same as site-language
- Loops through each data-provider-options and translates the labels and descriptions

### TODO
In order to get this working, we need the spanish translations from the translators.
Once they are there, add the english strings and translations to i18n/es.php folder.

Test checklist:
In the client
Start with english
- go to settings-> 'dataproviders'
- [ ] All texts should be in English
- Switch site-language to Spanish
- go to settings-> dataproviders
- [ ] All texts should be in Spanish


Fixes ushahidi/platform# .

Ping @ushahidi/platform
